influxdb3/tests/server/query.rs

```diff
@@ -270,6 +391,130 @@ async fn api_v3_query_influxql() {
+                "params": {
+                    "host": "b",
+                    "usage": 0.60,
+                },
+                "format": "pretty",
+            }))
+            .send()
+            .await
+            .unwrap()
+            .text()
+            .await
+            .unwrap();
+
+        assert_eq!(
+            "+------------------+---------------------+------+---------+-------+\n\
+            | iox::measurement | time                | host | region  | usage |\n\
+            +------------------+---------------------+------+---------+-------+\n\
+            | cpu              | 1970-01-01T00:00:03 | b    | us-east | 0.7   |\n\
+            | cpu              | 1970-01-01T00:00:04 | b    | us-east | 0.8   |\n\
+            +------------------+---------------------+------+---------+-------+",
+            resp
+        );
+    }
+
+    // Use a GET request
+    {
+        let params = serde_json::to_string(&json!({
+            "host": "b",
+            "usage": 0.60,
+        }))
+        .unwrap();
+        let resp = client
+            .get(&url)
+            .query(&[
+                ("db", "foo"),
+                (
+                    "q",
+                    "SELECT * FROM cpu WHERE host = $host AND usage > $usage",
+                ),
+                ("format", "pretty"),
+                ("params", params.as_str()),
+            ])
+            .send()
+            .await
+            .unwrap()
+            .text()
+            .await
+            .unwrap();
+
+        assert_eq!(
+            "+------------------+---------------------+------+---------+-------+\n\
+            | iox::measurement | time                | host | region  | usage |\n\
+            +------------------+---------------------+------+---------+-------+\n\
+            | cpu              | 1970-01-01T00:00:03 | b    | us-east | 0.7   |\n\
+            | cpu              | 1970-01-01T00:00:04 | b    | us-east | 0.8   |\n\
+            +------------------+---------------------+------+---------+-------+",
+            resp
+        );
+    }
+
+    // Check for errors
+    {
+        let resp = client
+            .post(&url)
+            .json(&json!({
+                "db": "foo",
+                "q": "SELECT * FROM cpu WHERE host = $host",
+                "params": {
+                    "not_host": "a"
+                },
+                "format": "pretty",
+            }))
+            .send()
+            .await
+            .unwrap();
+        let status = resp.status();
+        let body = resp.text().await.unwrap();
+
+        // TODO - it would be nice if this was a 4xx error, because this is really
+        //   a user error; however, the underlying error that occurs when Logical
+        //   planning is Datafusion::Internal, and is not so convenient to deal with.
+        //
+        //   I filed https://github.com/apache/arrow-datafusion/issues/9738 to see
```

Comment:
I feel for both of these comments it makes sense to remove the use of I as anyone who comes to read the code won't know who it is. I know it's you but they won't